lib/pbench/test/unit/server/test_user_auth.py

```diff
@@ -298,6 +298,33 @@ def test_update_user(client, server_config):
             assert response.status_code == 200
             assert data_login["message"] == "Successfully logged in."
 
+    @staticmethod
+    def test_external_token_update(client, server_config):
+        """ Test for external attempt at updating auth token"""
+        with client:
+            resp_register = register_user(
+                client,
+                server_config,
+                username="username",
+                firstname="firstname",
+                lastname="lastName",
+                email="user@domain.com",
+                password="12345",
+            )
+            data_register = resp_register.json
+            assert data_register["message"] == "Successfully registered."
```

Comment:
For the record, I still don't like including these `"message"` tags in successful responses. On errors, an error document with a more specific and informative message than the standard HTTP response code is good; I'd prefer that a successful response just be the resource we need to return without embellishment/complication. And saying "Successfully registered" doesn't add anything useful over the 200 status. Presumably the caller knows they were registering a user.

Reply:
Indeed but sometime we dont have any resource to return e.g register, logout, delete operations. In these cases should we just return the status code without anything else? (It does actually make sense because sending message string along with status code is a bit redundant). I initially did this to have consistent responses from all the operations because on login, get, put operations we return jsonified resource along with a status code so on operations that does not have anything to return just jsonify simple message and return along with a status code. However, I can certainly just do `return ' ', 201/200` if that make sense.

Reply:
Nothing says you need to return a body if there's nothing to return.

Normal REST convention for POST/PUT is to return the resource (in the body) or a reference to it (usually via `Location` header with a URI that you can `GET`). While that's not a bad thing to consider, I'm not sure it's critically important at this point. No reason for successful logout or a delete to return anything, because the HTTP status is enough.